Apps/W1/HybridSL/app/src/pages/IntelligentCloudExtension.PageExt.al

```diff
@@ -0,0 +1,61 @@
+pageextension 42000 "SL Intelligent Cloud Extension" extends "Intelligent Cloud Management"
```

Comment:
I believe that this is the old UI and that it is not needed. If not, we also need conditional visibility if the SL migration is enabled or not.

Reply:
This page has been removed.